src/components/LayerInformationDialog/LayerInformationDialog.vue

```diff
@@ -10,7 +10,38 @@
       <v-card>
         <v-card-title class="text-h5"> {{ layer.name }} </v-card-title>
 
-        <v-card-text v-html="layer.description" />
+        <v-card-text>
+          <dl class="layer-information-dialog__list">
```

Comment:
I'm using the same structure than on [this viewer](https://viewer.openearth.nl/ihm-viewer/?layers=85222873&layerNames=Maritieme%20grenzen%20-%20Noordzee%20%28INSPIRE%20aanmerking%29.)